omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/PatientIdentifierController1_9Test.java

```diff
@@ -77,6 +78,16 @@ public void shouldListAllPatientIdentifiersForAPatient() throws Exception {
 	}
 	
 	@Test
+	public void shouldListAllPatientIdentifiersWithVoidedIdentifiersForAPatient() throws Exception {
+		String urlString = getURI();
+		urlString += "?includeAll=true";
+		MockHttpServletRequest req = request(RequestMethod.GET, urlString);
```

Comment:
Can you do the includeAll parameter as here? https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/VisitAttributeTypeController1_9Test.java#L118-L119